app/src/org/commcare/views/widgets/RecordingFragment.java

```diff
@@ -63,7 +64,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         prepareButtons();
         prepareText();
         setWindowSize();
-        fileName = Environment.getExternalStorageDirectory().getAbsolutePath() + FILE_EXT + listener.getFileExtension();
+        fileName = Environment.getExternalStorageDirectory().getAbsolutePath() + FILE_BASE + listener.getFileExtension() + FILE_EXT;

```

Comment:
Looking at the implementation of getFileExtension() in CommCareAudioWidget, it looks like getFileExtension is a really misleading method name. Isn't it just providing some unique identifier for the given audio widget? If that's correct, can we rename it?